packages/common/driver-definitions/api-report/driver-definitions.alpha.api.md

```diff
@@ -647,9 +647,9 @@ export interface ITrace {
 export interface ITree {
     // (undocumented)
     entries: ITreeEntry[];
-    groupId?: string;
+    groupId?: string | undefined;
     id?: string;
-    unreferenced?: true;
+    unreferenced?: true | undefined;
```

Comment:
In general I think this is OK (it reflects what we're actually doing today), but it might be better to instead enforce the original typing - I believe this type is coming/going over the wire and "present but undefined" doesn't survive JSON.stringify/parse roundtrip.

I think OK to proceed as-is unless any other reviewer has a strong opinion.

Reply:
Yes, I agree. We should not add undefined in those.

Reply:
There are some places in the code that do "groupId: undefined", so I assume that's the reason we see these changes.
So, it feels like this PR reflects what is actually done today (i.e. it does not change behavior), but maybe there is an opportunity to make it better (maybe as a separate PRs)?

Reply:
yep. it does not change the behavior. 

i'm interested in identifying potential improvements from this PR and create work items for them.